var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -13,19 +13,34 @@ class Fms(CMakePackage):
     system models."""
 
     homepage = "https://github.com/NOAA-GFDL/FMS"
-    url      = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2021.02.01.tar.gz"
+    url      = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2022.02.tar.gz"
+    git      = "https://github.com/NOAA-GFDL/FMS.git"
 
-    maintainers = ['kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett']
+    maintainers = ['kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett', 'rem1776']
 
-    variant('64bit', default=True, description='64 bit?')
-    variant('gfs_phys', default=True, description='Use GFS Physics?')
-    variant('openmp', default=True, description='Use OpenMP?')
+    variant('64bit', default=True, description='64 bit')
+    variant('gfs_phys', default=True, description='Use GFS Physics')
+    variant('openmp', default=True, description='Use OpenMP')
+    variant('quad_precision', default=True, description='quad precision reals')
+    variant('yaml', default=False, description='yaml input file support(requires libyaml)')
+    variant('constants', values=str, default='GFDL', description='Build with <X> constants parameter definitions')
```

Comment:
These options need equivalent CMake args. If `yaml`, for example, is enabled in Spack that option is never passed to CMake.

Reply:
The cmake args for those variants are added with 1710637 along with some other improvements to the options